internal/stage_positive_character_groups.go

```diff
@@ -14,6 +14,11 @@ func testPositiveCharacterGroups(stageHarness *test_case_harness.TestCaseHarness
 			Input:            "efgh",
 			ExpectedExitCode: 1,
 		},
+		{
+			Pattern:          "[abcd]",
```

Comment:
THe pattern in all three cases is fixed here. Let's try to add a bit of randomness in the patterns? At the very least we should be using slightly different patterns. Ideally, we'd use random letters each time too. 

No stage should be easy to hardcode - this is super simple to hardcode